engine/evarutil.mli

```diff
@@ -10,6 +10,7 @@ open Names
 open Term
 open Evd
 open Environ
+open EConstr
```

Comment:
opening Econstr to hijack constr (removing its qualification below) is not making the mli more readable IMO